stylesheets/components/avatars/avatars.css

```diff
@@ -0,0 +1,48 @@
+@import "/stylesheets/index.css";
+
+.avatar-sm {
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+}
+
+.avatar-md {
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+}
+
+.avatar-lg {
+    width: 70px;
+    height: 70px;
+    border-radius: 50%;
+}
+
+.squared{
+    border-radius: 2px;
+
+}
+
+.sized-avatars{
+    background: #f8ebe7;
```

Comment:
using color variables will be good

Reply:
Thanks for pointing this out, Deekshith. Looks like I missed using them here. Will update it.